src/index.html

```diff
@@ -8,16 +8,13 @@
   </head>
   <body>
     <form>
```

Comment:
Great job updating the logo source to use a scalable vector graphic! Also, nice work on updating the email input to use the correct HTML type for email validation, as well as updating the password input to hide password visibility. One suggestion would be to remove the commented out TODOs in the HTML code. Keep up the good work!